### PR TITLE
Add drush script to calculate screen hash

### PR DESCRIPTION
**Pull request by HelenaEksler**
#303

### CODE REVIEW

**Comment by HelenaEksler** [comment]
selection_042

File: db_update.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+ROOT=$(pwd)
+cd $ROOT
+cd $ROOT/www
+
+echo "Importing database"
+drush sql-drop -y
+pv ~/Downloads/shoov_live_2015-11-28T10-00-00_UTC_database.sql.gz | gunzip | drush sql-cli
```
Comment (amitaibu) [inline db_update.sh]: these should be variables, and the file should be `db_import.exacmple.sh`

File: db_update.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+echo "Sanitize mails and github tokens"
+drush sql-query 'UPDATE users SET mail = CONCAT(mail, ".test");'
+drush sql-query 'UPDATE field_data_field_github_access_token SET field_github_access_token_value = "helena" WHERE entity_id NOT IN (SELECT uid FROM users WHERE name = "HelenaEksler");'
```
Comment (amitaibu) [inline db_update.sh]: variables (e.g. for `helena`)

**Comment by HelenaEksler** [comment]
Changed script to show total amount of items that were deleted.

@amitaibu PR is ready for review.

**Comment by HelenaEksler** [comment]
@amitaibu IS it ok, that after we are making the hash field required - we can't create screenshot manually (node/create/sreenshot), only using restful. And also, maybe add function to set hash field during migration?

**Comment by amitaibu** [comment]
>  we can't create screenshot manually 

Why not - you can feel the hash manually, no?

**Comment by amitaibu** [comment]
Migrate could feel the hash with dummy data.

**Comment by HelenaEksler** [comment]
The point is that hash won't be correct.

**Comment by HelenaEksler** [comment]
Actually, it doesn't matter, you're right

**Comment by amitaibu** [comment]
Merged. I've changed the location of the migrate script.

**Pull request closed without merging**